geniza/corpus/management/commands/import_data.py

```diff
@@ -571,3 +608,306 @@ def update_document_id_sequence(self):
         cursor.execute(
             "SELECT setval('corpus_document_id_seq', max(id)) FROM corpus_document;")
 
+    re_url = re.compile(r'(?P<url>https://[^ ]+)')
+
+    # ignore these entries in the editor field:
+    editor_ignore = [
+        'awaiting transcription',
+        'transcription listed on fgp',
+        'transcription listed on fgp, awaiting digitization on pgp',
+        'transcription listed in fgp, awaiting digitization on pgp',
+        'source of transcription not noted in original pgp database',
+        'yes',
+        'partial transcription listed in fgp, awaiting digitization on pgp.',
+        'partial transcription listed in fgp, awaiting digitization on pgp',
+        'transcription (recto only) listed in fgp, awaiting digitization on pgp',
+        'transcription in progress',
+        'awaiting transcription (htr)',
+        'transcription (verso only) listed in fgp, awaiting digitization on pgp',
+    ]
+
+    # re_docrelation = re.compile(r'^(. Also )?Ed. (and transl?.)? ?',
+    re_docrelation = re.compile(r'^(also )?ed. ?(and transl?. ?)?',
+                                flags=re.I)
+
+    # notes that may occur with an edition
+    # - full transcription listed/awaiting ...
+    # - with (minor) ..
+    # - with corrections
+    # - multiword parenthetical at the end of the edition
+
+    re_ed_notes = re.compile(
+        r'[.;] (?P<note>(' +
+        r'(full )?transcription (listed|awaiting|available).*$|' +
+        r'(with )?minor|with corrections).*$|' +
+        r'awaiting digitization.*$|' +
+        r'; edited (here )?in comparison with.*$|' +
+        r'[(]?see .*$|' +
+        r'(\(\w+ [\w ]+\) ?$))',
+        flags=re.I)
+
+    # regexes to pull out page or document location
+    re_page_location = re.compile(
+        r'[,.] (?P<pages>((pp?|pgs)\. ?\d+([-–]\d+)?)|(\d+[-–]\d+))\.?',
+        flags=re.I)
+    re_doc_location = re.compile(
+        r'(, )?\(?(?P<doc>(Doc. #?|#)([A-Z]-)?\d+)\)?\.?',
+        flags=re.I)
+    # \u0590-\u05fe = range for hebrew characters
+    re_goitein_section = re.compile(
+        r' (?P<p>(\d+?[\u0590-\u05fe]|[\u0590-\u05fe]\d+)[\u0590-\u05fe]?)',
+        flags=re.I)
```

Comment:
nice!!! I was wondering if something like this was possible; really cool solution

Reply:
yes! found a good resource online that had the range